### PR TITLE
Django Home Page new test cases 

### PR DESCRIPTION
**Pull request by kevdouglass**
Added Django Test Case for

 - (User and Http Responses Valid response)
- Made 'Media' Directory visible for serving dynamic img's and video's with a relative path.
- script for camerPortScan.py that will be used to scan ports to check that user has a visible camera for OpenCV script.

### CODE REVIEW

File: osCam/osCam/urls.py [hunk 1]
```diff
@@ -15,11 +15,19 @@
...
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('core.urls'), name='core'),
     path('', include('user.urls'), name='user'),
     path('settings/', include('userconfig.urls'), name='settings'),
     path('stream/', include('streamin.urls'), name='stream'),
-]
+]+ static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+# urlpatterns += urlpatterns('',
```
Comment (melippmann) [inline osCam/osCam/urls.py:29]: remove old commented out code from PRs

File: osCam/templates/core/home.html [hunk 2]
```diff
@@ -41,7 +41,8 @@ <h2>cameras</h2>
                     {% if user.is_authenticated %}
                         
                     <div class="container-fluid">
-                        <img src="{% url 'feed' %}" class="vid-feed">
+                        <img src="media/video/vault.jpg" class="vid-feed">
+                        <!-- <img src="../../video/vault.jpg"> -->
```
Comment (melippmann) [inline osCam/templates/core/home.html:45]: remove comment

**Review by melippmann — CHANGES_REQUESTED** [review]
generally looks good, please clean up linting and let remove commented out code, notify me when this is done and I will accept the PR.

**Comment by DJ-IRL** [comment]
pending meher's OK this all looks good to me

File: osCam/core/tests.py [hunk 3]
```diff
@@ -5,3 +5,73 @@
...
+# Create your tests here.
+class smokeTest(TestCase):
+    def test_smoke(self):
+        self.assertEqual(1, 1)
+
+class SetupMotionDetectCamera(TestCase):
+    
+    objects = {
+        #Constants
+        'PORTNUM': 0,
```
Comment (melippmann) [inline osCam/core/tests.py:34]: please remove port num

**Review by melippmann — APPROVED** [review]

**Pull request merged**